web-client/src/environments/environment.ts

```diff
@@ -32,8 +32,12 @@ export const environment: Environment = {
   commissionPercentage: 0.01,
   tokenIssuer: 'rpJv16Qmn2rQP6UC6UFsNRnVy5arkQihPP',
   tokenSymbol: 'FOO',
-  xrpIssuer: 'rpJv16Qmn2rQP6UC6UFsNRnVy5arkQihPP',
+  xrpIssuer: 'rf9vKLZVsozsaUgU533W4BUj87TgzTqyQE',
   hideXrpBalance: false,
+  autofundXrp: true,
+  autofundXrpAmount: 11,
+  autofundAccountPin: "K3VG-M@YQF9gzjq.DGW!",
```

Comment:
convert double quotes to single quotes for both `autofundAccountPin` and `autofundAccountPin`